src/features/unified-share-modal/messages.js

```diff
@@ -291,11 +291,6 @@ const messages = defineMessages({
         description: 'Description of an open shared link',
         id: 'boxui.unifiedShare.peopleWithLinkDescription',
     },
-    peopleWithLinkSignedInRequiredDescription: {
```

Comment:
are we sure we/someone else won't need this in the future for any reason? since deleting the message deletes the translation; and i assume Box pays for each translation that needs to be generated